### PR TITLE
Ticket 42:  refresh btn + generic modal

### PR DESCRIPTION
**Pull request by maximFeldman1**
create refresh button for get api request and generic modal for edit and create product

Screen Shot 2023-01-09 at 12 22 25
Screen Shot 2023-01-09 at 12 21 59

### CODE REVIEW

File: src/i18n/translation/products/table/dropdown/modal/en.ts [hunk 1]
```diff
@@ -2,4 +2,5 @@ export const modal = {
   edit: "Edit",
   delete: "Delete",
   options: "Options",
+  editTitle: "Edit Product",
```
Comment (amitmarko) [inline src/i18n/translation/products/table/dropdown/modal/en.ts]:
```suggestion
  editProduct: "Edit Product",
```

File: src/i18n/translation/products/toast/en.ts [hunk 2]
```diff
@@ -9,4 +9,5 @@ export const toast = {
     errorMessage: "Failed",
     faild: "Your Delete Failed",
   },
+  wantToDelete: "Are you sure that you want to delete product?",
```
Comment (amitmarko) [inline src/i18n/translation/products/toast/en.ts]: Move to delete toast above and change to:
`message: "Are you sure that you want to delete product?"`

File: src/i18n/translation/products/table/dropdown/modal/en.ts (hunk 1 above)
Comment (maximFeldman1) [reply src/i18n/translation/products/table/dropdown/modal/en.ts]: Fixed

File: src/i18n/translation/products/toast/en.ts (hunk 2 above)
Comment (maximFeldman1) [reply src/i18n/translation/products/toast/en.ts]: Fixed

File: src/Pages/CreateProduct/components/ProductForm/index.tsx [hunk 3]
```diff
@@ -14,14 +14,21 @@ interface IProps {
...
   onCancel,
   onSubmit,
   onClickBack,
+  visible,
 }: IProps) => {
+  const WrapperBtn = styled.div`
+    position: ${!visible ? "fixed" : "none"};
+    bottom: ${!visible ? "30px" : "0"};
+    right: ${!visible ? "50px" : "0"};
+  `;
```
Comment (itairom-appsforce) [inline src/Pages/CreateProduct/components/ProductForm/index.tsx]: maybe extract it outside & pass prop , something like this: 

interface WrapperBtnProps {
  visible?: boolean;
}

image

Comment (itairom-appsforce) [reply src/Pages/CreateProduct/components/ProductForm/index.tsx:31]: const WrapperBtn = styled.div<WrapperBtnProps>`
  position: ${({visible}) => (visible ? "fixed" : "none")};
`;

Comment (maximFeldman1) [reply src/Pages/CreateProduct/components/ProductForm/index.tsx]: Fixed

**Pull request merged**